whiteBoarding/bubbleSort/whiteboardingcode

```diff
@@ -0,0 +1,29 @@
+// Whiteboarding Question, Monday, Oct. 30, 2018
+// Given an array, sort the array using bubbleSort and then return the array.
+
+function myBubble[arr] => {
+    var inspect = ( );
+    var arraylenth = arr.lenth;
+    var arrIndex = arr;
+    if (inspect = false;
```

Comment:
this is not a comparator this is a setting.  `inspect == false` would be a conditional